modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/rest-openapi.yaml

```diff
@@ -422,6 +445,51 @@ info:
     version: v1.0
 openapi: 3.0.1
 paths:
+    "/asset-libraries/taxonomy-vocabularies/by-spaces":
```

Comment:
@ctang3 what does by spaces mean?  doesn't asset libraries already mean spaces?

Reply:
It does, Asset Library = Space. I added the `by-spaces` thinking it makes a distinction that its for CMS 2.0 and associates the vocabulary with multiple Asset Libraries. 

Should I just omit that and go with something like `/asset-libraries/taxonomy-vocabularies`? I realize now that the terminology should probably match the DTO over matching the frontend terms the User sees?